pkg/daemon/certificate_writer.go

```diff
@@ -186,7 +214,71 @@ func (dn *Daemon) syncControllerConfigHandler(key string) error {
 		}
 		klog.Infof("Certificate was synced from controllerconfig resourceVersion %s", controllerConfig.ObjectMeta.ResourceVersion)
 
+		if cmErr == nil {
+			cmd := exec.Command("systemctl", "stop", "kubelet")
+			errOut := &bytes.Buffer{}
+			cmd.Stderr = errOut
+			err = cmd.Run()
```

Comment:
suggestion: Unless we really need the contents of `stdout` or `stderr` to be separated (e.g., we want to parse it for something else), do `cmd.CombinedOutput()` instead. It will combine the output together so that you can emit everything at once.